Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.8-slim-buster
+FROM python:3.9-slim
 WORKDIR /usr/src/app
 COPY requirements_old.txt ./
 RUN pip3 install --upgrade pip
 RUN pip3 install --no-cache-dir -r requirements.txt
 COPY . .
```

Comment:
It would be nice to add a separate non-privileged user here and run the application on his behalf

Reply:
Tried to fix it in a new commit, but perhaps an example is needed

Reply:
THX. Checked and added changes